Observer/Order/Save.php

```diff
@@ -132,7 +132,9 @@ public function execute(\Magento\Framework\Event\Observer $observer)
                 ->create()
                 ->load($quoteId, NostoCustomer::QUOTE_ID);
 
-            $orderService = new \NostoServiceOrder($nostoAccount);
+            if ($nostoAccount) {
```

Comment:
Can you please change this to ` if ($nostoAccount instanceof \NostoAccountMetaInterface) {` and wrap the whole try catch block inside that if block